frontend-student/src/components/coursePage/ModuleItem.jsx

```
@@ -35,24 +42,57 @@ export default function ModuleItem({
-            onSelect={() => handleVideoSelect(video, mIndex + 1, modIndex + 1, vIndex + 1)}
-            indexPath={`${mIndex + 1}.${modIndex + 1}.${vIndex + 1}`}
-          />
-        ))}
+        {module.moduleItems?.map((item, itemIndex) => {
+          if (item.quiz) {
+            quizCount++;
+          }
+
+          const isPreviousMilestone =
+            !isCurrentMilestone &&
+            mIndex < parseInt(selectedItem?.milestoneNumber || 0) - 1;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)